src/yunohost/certificate.py

```diff
@@ -810,9 +810,20 @@ def _domain_is_accessible_through_HTTP(ip, domain):
 
 
 def _name_self_CA():
-    cert = crypto.load_certificate(
-        crypto.FILETYPE_PEM, open(SELF_CA_FILE).read())
-    return cert.get_subject().CN
+    ca_conf = os.path.join(SSL_DIR, "openssl.ca.cnf")
```

Comment:
Seems a little wrong to me to get the info you're looking for from this file.
Shouldn't it be interpreted from the cert itself ?

Is it what you are looking for ? :
```
openssl  x509 -text -noout -in /usr/share/yunohost/yunohost-config/ssl/yunoCA/ca/cacert.pem
```
then grep for ```Subject : CN=domain.com```